corehq/apps/export/views/download.py

```diff
@@ -537,3 +540,18 @@ def add_export_email_request(request, domain):
     else:
         EmailExportWhenDoneRequest.objects.create(domain=domain, download_id=download_id, user_id=user_id)
     return HttpResponse(ugettext_lazy('Export e-mail request sent.'))
+
+
+@method_decorator(login_and_domain_required, name='dispatch')
+class DownloadDETSchemaView(View):
+    urlname = 'download-det-schema'
+
+    def get(self, request, domain, export_instance_id):
+        export_instance = get_properly_wrapped_export_instance(export_instance_id)
+        assert domain == export_instance.domain
+        output_file = BytesIO()
+        generate_from_export_instance(export_instance, output_file)
+        output_file.seek(0)
+        response = HttpResponse(output_file.read(), content_type='application/vnd.ms-excel')
```

Comment:
nit: `HttpResponse` accepts a file-like object, right?
```suggestion
        response = HttpResponse(output_file, content_type='application/vnd.ms-excel')
```